plugins/player-activity-tracker.js

```diff
@@ -1,14 +1,14 @@
 // @author         breunigs
 // @name           Player activity tracker
 // @category       Layer
-// @version        0.12.2
+// @version        0.12.3
 // @description    Draw trails for the path a user took onto the map based on status messages in COMMs. Uses up to three hours of data. Does not request chat data on its own, even if that would be useful.
 
 /* exported setup, changelog --eslint */
 
 var changelog = [
   {
-    version: '0.12.2',
+    version: '0.12.3',
```

Comment:
Imho there's no need to change version here as only the changelog entry was added, nothing more